.gitignore

```diff
@@ -3,6 +3,7 @@
 dist/
 deno.lock
 node_modules/
+package-lock.json
```

Comment:
For your information, you have to use pnpm instead npm when you work on Fedify.

Reply:
removed https://github.com/fedify-dev/fedify/pull/281/commits/eb9a205c7fa8d25e21b309753e1741d4241ebf64 👍